jest/mocks/dataMocks/users/getUsers.js

```diff
@@ -0,0 +1,49 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import faker from 'faker'
+import moment from 'moment'
+
+import { getRandomElement, generateArrayOfObjects } from '../../../helpers'
+
+import { userStatusMap, userRolesMap } from '../../../../src/constants/app'
+
+export const getUser = dataToOverwrite => {
+  const firstName = faker.name.firstName()
+  const lastName = faker.name.lastName()
+
+  return {
+    avatar: faker.image.imageUrl(),
+    displayName: `${firstName} ${lastName}`,
+    email: faker.internet.email(),
+    firstName,
+    hideEmailField: faker.datatype.boolean(),
+    highlighted: `${firstName} <em>${lastName}</em>`,
+    id: faker.datatype.number(),
+    isActive: faker.datatype.boolean(),
+    lastMessageTimestamp: moment(faker.datatype.datetime()).format(
+      'YYYY-MM-DDTHH:mm:ss.SSSSSSZ',
+    ),
+    lastName,
+    phoneNumber: faker.phone.phoneNumber(),
+    pm: faker.datatype.number(),
+    role: getRandomElement(Object.values(userRolesMap)),
+    section: '#',
```

Comment:
I have no idea, copied the full backend response. From what I saw, section always returns '#'

Reply:
discussed with riccardo, this was for a phonebook-like feature on iOS. if a user jump to all contacts starting with "G", you can ask the backend about all contacts in section "G". the RPC will be changed now because this is deprecated.

Reply:
Ok, I'll remove it

Reply:
Just pushed updated